### PR TITLE
chore(e2e): adjust the scripts to make it easier to run atlas cloud e2e tests locally

### PR DESCRIPTION
**Pull request by gribnoysup**
Small follow up to the new e2e capabilities to make it a bit easier to run these locally if needed. I adjusted the implies check in yargs parsing so that you can keep these env in your shell env config without triggering the implies check when not running the tests for cloud (you still need to explicitly provide the `--test-atlas-cloud` flag to trigger the test and it's still required to provide all the other options in that case, just not otherwise), also added some detailed instructions for what needs to be manually setup before running those, should be helpful also if we ever need to re-do the user creation

### CODE REVIEW

File: .evergreen/start-atlas-cloud-cluster.sh [hunk 1]
```diff
@@ -17,23 +68,43 @@ function atlascli() {
...
 
 echo "Waiting for the deployment to be provisioned..."
-atlascli clusters watch "$ATLAS_CLOUD_TEST_CLUSTER_NAME"
+atlascli clusters watch $ATLAS_CLUSTER_NAME
 
 echo "Getting connection string for provisioned cluster..."
-ATLAS_CLOUD_TEST_CLUSTER_CONNECTION_STRING_JSON="$(atlascli clusters connectionStrings describe $ATLAS_CLOUD_TEST_CLUSTER_NAME -o json)"
+CONNECTION_STRINGS_JSON="$(atlascli clusters connectionStrings describe $ATLAS_CLUSTER_NAME -o json)"
+
+export COMPASS_E2e_ATLAS_CLOUD_SANDBOX_CLOUD_CONFIG=$(
```
Comment (mabaasit) [inline .evergreen/start-atlas-cloud-cluster.sh:98]:
```suggestion
export COMPASS_E2E_ATLAS_CLOUD_SANDBOX_CLOUD_CONFIG=$(
```

Comment (mabaasit) [reply .evergreen/start-atlas-cloud-cluster.sh:98]: As we are passing this to `test-web-sandbox-atlas-cloud` evergreen function (as `qa`), should we check if that's set?

Comment (mabaasit) [reply .evergreen/start-atlas-cloud-cluster.sh:98]: Or maybe remove it from there as we are now using `MCLI_OPS_MANAGER_URL` to determine this?

Comment (gribnoysup) [reply .evergreen/start-atlas-cloud-cluster.sh:98]: Yeah, can drop it from evergreen config, forgot about that, makes sense 👍

Comment (gribnoysup) [reply .evergreen/start-atlas-cloud-cluster.sh]: Done in e284976f3aa1516ebf4b8c8a75d27ba8530b9a9b with the typo fix, thanks for spotting!

**Review by mabaasit — APPROVED** [review]

**Pull request merged**